www/src/context/SettingsContext.jsx

```diff
@@ -11,12 +11,27 @@ const defaultValue = {
 const SettingsContext = createContext(defaultValue);
 
 const SettingsContextProvider = ({ children }) => {
+  // gatsby does not have access to the localStorage during the build (and first render)
+  // so sadly we cannot initialize theme with value from localStorage
   const [theme, setTheme] = useState('openedx-theme');
   const [showSettings, setShowSettings] = useState(false);
 
+  const handleThemeChange = (e) => {
+    setTheme(e.target.value);
+    localStorage.setItem('theme', e.target.value);
```

Comment:
nit: should we use `global.localStorage`?

Reply:
Yeah, this should be better, updated, thanks!